<h1 style='font-size:40px'> HTTPS Setup with Kubernetes</h1>

<h2 style='font-size:30px'> 
    HTTPS Setup Overview
</h2>
<div> 
    <ul style='font-size:20px'> 
        <li>
        </li>
    </ul>
 </div>

In [11]:
! git add .
! git commit -m 'Iniciar seção 17'
! git push

[master 79cebf1] Aula 286
 10 files changed, 29 insertions(+), 42 deletions(-)
Enumerating objects: 35, done.
Counting objects: 100% (35/35), done.
Delta compression using up to 24 threads
Compressing objects: 100% (20/20), done.
Writing objects: 100% (20/20), 2.53 KiB | 2.53 MiB/s, done.
Total 20 (delta 16), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (16/16), completed with 14 local objects.
To https://github.com/felipesveiga/docker.git
   b0fc7f5..79cebf1  master -> master


<p style='color:red'> Vi Aula 286 a 290; Iniciar seção 17</p>

<p style='color:red'> Visite a Aula 307, caso deseje deletar o seu cluster!</p>